### DISCUSSION TITLE
WANDB support

### DISCUSSION THREAD

**Discussion by CHeesetoucher2** [trackmania-rl/tmrl#24] [Ideas]
Would be cool to visualize data from runs on wandb, such as checkpoint times, distance drives, graph over speed, time to finish map etc etc etc

**Comment by yannbouteiller** [comment]
Hi, there is already some basic support for wandb, but probably we can implement a more clever way of sending environment-specifific metrics, like putting these in a special entry of the info dict ? 

Distance drive and time to finish map can already be inferred from the reward graph and the episode duration graph.

**Comment by CHeesetoucher2** [comment]
Oh missed that, but would be cool with more stats
